js/src/ui/MethodDecoding/methodDecoding.css

```diff
@@ -16,6 +16,7 @@
 */
 
 .method {
+  font-family: 'Roboto Mono' !important;
```

Comment:
Was discussing it earlier today, really missing Sass variables, these things aren't very DRY. Should probably fix it all-over. Which brings me to the next point, fonts should be bundles as part of Webpack. (Tomasz did it somewhere in one of his apps, everything should be served with Parity, fast, no connection needed to play locally)

https://github.com/ethcore/parity/issues/2344